crates/ark-contracts/arkchain/src/order/types.cairo

```diff
@@ -154,14 +155,28 @@ impl Felt252TryIntoOrderStatus of TryInto<felt252, OrderStatus> {
 struct ExecutionInfo {
```

Comment:
Should be changed to `FulfillInfo` to be consistent with the vocabulary we've chosen.